package.json

```diff
@@ -22,17 +22,13 @@
   },
   "devDependencies": {
     "chai": "^4.2.0",
-    "coffee-script": ">=1.8.x",
```

Comment:
🤔 I know this is just temporary, but I think we should add @goodeggs/toolkit (linting and prettier formatting) in this PR. See the TS guide.

Then the next PR can just focus on porting to TS?